### ISSUE TITLE
Define the Machine Learning Component (RNN)

### ISSUE THREAD

**Issue by Startonix** [Startonix/ModularAI-#79]
Using TensorFlow to define an RNN that processes the output from the neuromorphic network.

import tensorflow as tf

def create_rnn_model(input_shape):
    model = tf.keras.Sequential([
        tf.keras.layers.SimpleRNN(50, activation='relu', input_shape=input_shape),
        tf.keras.layers.Dense(10, activation='softmax')
    ])
    model.compile(optimizer='adam', loss='sparse_categorical_crossentropy', metrics=['accuracy'])
    return model
